src/renderer/components/PluginManager/PluginManagerTable.vue

```diff
@@ -20,21 +20,22 @@
 
         <div
           v-else-if="data.column.field === 'title'"
+          class="flex items-center"
         >
           <span
             class="whitespace-no-wrap cursor-pointer"
             @click="emitShowDetails(data.row)"
           >
             {{ data.row.title | truncate(30) }}
           </span>
+          <PluginManagerCheckmark v-if="data.row.isOfficial && !data.row.isGrant" />
```

Comment:
@alexbarnsley just to confirm... can a plugin be `isOfficial = true` and `isGrant = true`? If not, i'll remove the `!data.row.isGrant`

Reply:
I don't think so since they're just developed internally 👌